### ISSUE TITLE
VOC_20000.pth

### ISSUE THREAD

**Issue by yuan-001** [hfslyc/AdvSemiSeg#33]

**Comment by hfslyc** [comment]
please see the section: Testing on VOC2012 validation set with pretrained models

the VOC_20000.pth will appear after you performed training
